src/main.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { createStore, applyMiddleware } from 'redux';
+import ReactDom from 'react-dom';
+import { Provider } from 'react-redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import App from './component/app/app';
+import categoryReducer from './reducer/category';
```

Comment:
One of your big things you need to do for this lab is to import in your `combineReducers` module from `./reducer` and pass that into `createStore` in place of `categoryReducer`.